src/main/java/frc/robot/Robot.java

```diff
@@ -225,13 +226,16 @@ public void teleopPeriodic() {
 	 * Unused
 	 */
 	@Override
-	public void testInit() { }
+	public void testInit() {
+		NetworkTableInstance.getDefault().getTable("limelight").getEntry("ledMode").setNumber(3);
+	}
```

Comment:
try pulling Network table out into a variable. calling get on this a bunch of times may be slowing stuff down.